warehouse/tasks.py

```diff
@@ -119,6 +122,9 @@ def _after_commit_hook(self, success, *args, **kwargs):
         if success:
             super().apply_async(*args, **kwargs)
 
+    def on_failure(self, exc, task_id, args, kwargs, einfo):
```

Comment:
This is great! Thank you.

I'm actually wondering if we could get this into the Error logger so Sentry would capture them.

Reply:
To add more context here, @ewdurbin is talking about [this logging handler](https://github.com/pypa/warehouse/blob/9f3fb2f7cfb2563e714fc0a452e5ad79bcc11124/warehouse/logging.py#L68-L74) which is attached to the [root logger](https://github.com/pypa/warehouse/blob/9f3fb2f7cfb2563e714fc0a452e5ad79bcc11124/warehouse/logging.py#L78).

Reply:
I think maybe this is already happening as-is?